src/pages/cart-page/Main.jsx

```diff
@@ -0,0 +1,59 @@
+import img3 from "../../assets/images/Jeans1.JPG";
+export default function Main() {
```

Comment:
Give some space between importing components and your function

Reply:
done